### ISSUE TITLE
RadioInput groups are not Accessible

### ISSUE THREAD

**Issue by UncleClapton** [FuelRats/fuelrats.com#360]
### Describe the bug

RadioInput fails to be placed within tab order, making it inaccessible for screen readers

Input options should also have a focus state

### Expected behavior

I expect the thing to bloody work with tab controls

### Steps to reproduce

1. go to page with RadioInput control
2. try tabbing around!

**Comment by UncleClapton** [comment]
Confirmed issue on 2.13.1

**Comment by evertiro** [comment]
Since we're not displaying the actual input field, but rather treating its label as the input, the obvious solution would be to add a tabIndex to the label. This would work, but comes with its own set of issues - on its own, that change would trip up the no-noninteractive-tabindex rule. This _could_ be solved by giving the label a role (say, button), but would then cause another issue by tripping the no-noninteractive-element-to-interactive-role rule. To be honest, I'm not sure the best way to deal with this off hand.
